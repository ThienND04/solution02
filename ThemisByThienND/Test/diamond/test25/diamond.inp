14 11
911797 913921
563026 514501
731209 584641
772627 114201
288641 750001
234921 348561
613933 70459
966625 680833
502321 662361
96001 311158
845761 857761
227917 812481
961537 776501
49361 317501
