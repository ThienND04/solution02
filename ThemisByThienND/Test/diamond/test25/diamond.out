3599879 2917449
